components/tidb_query/src/rpn_expr/impl_time.rs

```diff
@@ -52,6 +53,21 @@ pub fn week_day(ctx: &mut EvalContext, t: &Option<DateTime>) -> Result<Option<In
     Ok(Some(i64::from(day)))
 }
 
+#[rpn_fn(capture = [ctx])]
+#[inline]
+pub fn to_days(ctx: &mut EvalContext, t: &Option<DateTime>) -> Result<Option<Int>> {
+    if t.is_none() {
+        return Ok(None);
+    }
+    let t = t.as_ref().unwrap();
+    if t.invalid_zero() {
```

Comment:
The original implementation is `if t.is_zero() {`. Do you have reasons to change it?

Reply:
I will change back later.